### PR TITLE
updated documentation

### PR DESCRIPTION
**Pull request by honeykyle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,13 +19,13 @@ The greatest difficulties encountered right now are caused by these issues that
...
- - UV channels of anime are subsampled (4:2:0), which means the color channels of 1080p anime are in fact 540p, thus there is a lack of 1080p ground truth for the UV channels.
+ - UV channels of anime are subsampled (4:2:0), which means the color channels of 1080p anime are 540p, thus there is a lack of 1080p ground truth for the UV channels.
  - Simulating H.264/H.265 compression artifacts (for analysis and denoising) is not trivial and is relatively time-consuming.
- - Due to the workflow of animation studios and their lack of time/budget, resampling artifacts of individual sprites are present in many modern anime.
- - Speed (preferably real-time) is paramount, since we do not want to re-encode video each time the algorithm improves. There is also less risk of permanently altering original content.
+ - Due to the workflow of animation studios and their lack of time/budget, resampling artifacts of individual sprites are present in much modern anime.
+ - Speed (preferably real-time) is paramount since we do not want to re-encode video each time the algorithm improves. There is also less risk of permanently altering the original content.
```
Comment (VihagChaturvedi) [inline README.md:25]: Line 24: "Much modern anime" is not correct. Rather, "A sizeable portion of modern anime" or similar sentences seem more legible imo.

File: README.md [hunk 2]
```diff
@@ -102,14 +102,14 @@ August 2019
...
-Remarkably, the proposed method does not use any machine-learning or statistical approach, and is tailored to content that puts importance to well defined lines/edges while tolerates a sacrifice of the finer textures. The proposed algorithm can be quickly described as an iterative algorithm that treats color information as a heightmap and 'pushes' pixels towards probable edges using gradient-ascent. This is very likely what learning-based approaches are already doing under the hood (eg. VDSR<sup>**[1]**</sup>, waifu2x<sup>**[2]**</sup>).
+We present a state-of-the-art high-quality real-time SISR algorithm designed to work with Japanese animation and cartoons that are extremely fast *(~3ms with Vega 64 GPU)*, temporally coherent, simple to implement *(~100 lines of code)*, yet very effective. We find it surprising that this method is not currently used 'en masse' since the intuition leading us to this algorithm is very straightforward.
```
Comment (VihagChaturvedi) [inline README.md:105]: Line 105: Lack of commas. It should be something along the lines of "state-of-the-art, high-quality and a real-time SISR algorithms...". I believe multiple algorithms are offered, so it should be in plural form

File: README.md [hunk 3]
```diff
@@ -131,19 +131,19 @@ There are no ground truths of 4K anime.
...
+### Note for those who think this is not an 'upscaling' algorithm.
 
 [Explanation](Upscaling.md)
 
 TL;DR
 
-Sharpening, De-Blurring and Super-Resolution are equivalent.  
-Anime4K can de-blur, and is equivalent to a SR algorithm.  
+Sharpening, De-Blurring, and Super-Resolution are equivalent.  
+Anime4K can de-blur, and is equivalent to an SR algorithm.
```
Comment (VihagChaturvedi) [inline README.md:147]: Line 147: Would be favourable to have "a Super-Resolution algorithm" rather than causing potential confusion
